config/tcs_config.toml

```diff
@@ -13,10 +13,8 @@
 # limitations under the License.
 
 [KvStorage]
-StoragePath = "config/Kv_Shared_tmp"
-StorageSize = "1 TB"
 # the remote version is of higher priority if enabled
-#remote_url = "http://localhost:9090"
+remote_url = "http://localhost:9090"
```

Comment:
remote_url is added in lmdb_config.toml ? So do we need it here

Reply:
In lmdb_config.toml remote_url is changed to bind. remote_url in config_tcs_config.toml is required for EM to connect to LMDB

Reply:
We should keep remote lmdb URL in single place to avoid duplication and ease of maintenance. Configuration which are common for modules can be kept in tcs_config.toml. So if remote lmdb url needs to be accessed by both lmdb and EM let's keep it only in tcs_config.toml

Reply:
No, listener is going to have separate config file, then both EM and listener is should have an entry for access lmdb.